src/dirtyfields/dirtyfields.py

```diff
@@ -28,7 +28,14 @@ def get_dirty_fields(self, check_relationship=False):
         new_state = self._as_dict(check_relationship)
         all_modify_field = {}
 
-        for key, value in new_state.iteritems():
+        # Python3
+        try:
+            items = new_state.items()
+        # Python2
+        except AttributeError:
+            items = new_state.iteritems()
```

Comment:
Doing this Error catching seems not to be the best solution, as some libraries, like `six` (https://pythonhosted.org/six/) have been created for this kind of problem.